backend/local/local.go

```diff
@@ -270,6 +270,11 @@ only useful for reading.
 				Value: cTime.String(),
 				Help:  "The last status change time.",
 			}},
+		}, {
```

Comment:
I agree. However, rclone will always use md5 if supported by the other backend, or sha1 etc. These new hashes will therefore never be used, unless they are the only ones the backend of the opposite side of a operation supports (or at least that it does not support the ones rclone prefers first). This means for local-local transfer md5 will always be used. For a SFTP remote one would have to make sure md5 and sha1 are disabled/unavailable if one wants to use the new hashes for a local-sftp transfer. This new config allows me to run a local-local copy operation and force rclone to use XXH3.

For me it was very useful for testing. Maybe not so relevant to include into master, and a more generic solution for hash preferences for backends supporting more than one would probably be better. I included it, however, because I was thinking the new hashes would otherwise have very limited use - I think only restricted to a sftp remote configured to support it and not md5/sha1.

Reply:
Ah, I see. Maybe a more generic solution like `--disable-hash MD5,SHA1` might work which could disable the hashes in `lib/hash`. Or maybe something like `--restrict-hash SHA1,XXH3`